### PR TITLE
Feature: Enable screenshots on capybara failure

### PR DESCRIPTION
**Pull request by bohme**
Asana-task: [Investigate: Do capybara screenshots work from the muffi template?](https://app.asana.com/0/1142794766483633/1156764737042457/f)

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -84,3 +84,9 @@ jobs:
         env:
           CC_TEST_REPORTER_ID: ${{ secrets.CC_TEST_REPORTER_ID }}
         run: ./cc-test-reporter after-build --exit-code $?
+      - name: Archive capybara artifacts
+        uses: actions/upload-artifact@v2
+        if: failure()
+        with:
+          name: capybara
+          path: tmp/capybara/
```
Comment (GeoffAbtion) [inline .github/workflows/ci.yml:92]: Do we need to document how to view the screenshots?

File: spec/support/capybara_screenshot.rb [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+require "capybara-screenshot/rspec"
```
Comment (GeoffAbtion) [inline spec/support/capybara_screenshot.rb:3]: This require looks odd

Comment (GeoffAbtion) [reply spec/support/capybara_screenshot.rb:3]: > This require looks odd

It's referring to something that's a part of the capybara-screenshot lib?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (bohme) [reply .github/workflows/ci.yml:92]: Github Actions Artifcats 🤔 
yea... it's probably a good idea 👍

File: spec/support/capybara_screenshot.rb (hunk 2 above)
Comment (bohme) [reply spec/support/capybara_screenshot.rb:3]: They tell us to do this on their readme so 🤷‍♀️

Comment (bohme) [reply spec/support/capybara_screenshot.rb:3]: it is part of `capybara-screenshot` yes 🤔

**Comment by jeppester** [comment]
@GeoffAbtion's concerns seem to have been resolved, I'm merging this

**Pull request merged**